.changeset/swift-kings-add.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-newrelic-dashboard': minor
```

Comment:
Think you can just keep this one changeset and remove the other, but it should be a `patch` version. :)